atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -86,7 +73,8 @@ def __init__(self, tasker, workflow, labels,
                               "com.redhat.build-host"),
                  aliases=None,
                  dont_overwrite_if_in_dockerfile=("distribution-scope",),
-                 info_url_format=None):
+                 info_url_format=None,
+                 equal_labels=[]):
```

Comment:
Shouldn't really declare default values as mutable objects such as lists. Instead, better to use `None` as the default value here and `or []` when setting the instance variable.